### ISSUE TITLE
"Add VMs" button doesn't change to "Add" on selecting Autoscale on VPC LB.

### ISSUE THREAD

**Issue by rajujith** [apache/cloudstack#7841]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI
~~~

##### CLOUDSTACK VERSION

~~~
ACS 4.18.0
~~~

##### CONFIGURATION

VPC

##### OS / ENVIRONMENT

NA
##### SUMMARY

On selecting the Autoscale "Yes" when adding a load balancer rule on VPC public IP the Button doesn't change to 'Add' from 'Add VMs'. Adding VMs is not expected for Autoscale load balancer rules. We can click on 'OK' on the instance selection form to add the rule but this is inconsistent with isolated networks and also does not make sense. 

##### STEPS TO REPRODUCE

~~~
1. Create a VPC.
2. Add a tier.
3. Go to VPC and click on the 'Public IP Addresses' tab.
4. Acquire a new public IP.
5. Click on the acquired 'Public IP Address' and select the 'load balancing' tab. 
6. Add a load balancing rule and select 'Autoscale' as 'Yes'. 
7. Observe that you are unable to add the Autoscale LB rule without going to the 'Add VMs form'.
~~~

##### EXPECTED RESULTS

~~~
Should be able to add an Autoscale load balancing rule in VPC without viewing the Add VMs form.
~~~

##### ACTUAL RESULTS

~~~
Unable to add an Autoscale load balancing rule in VPC without viewing the Add VMs form.
~~~

##### ISOLATED GUEST NETWORKS
![Screenshot 2023-08-10 at 2 54 58 PM](https://github.com/apache/cloudstack/assets/5229147/039d97df-ce4d-4d67-9d30-6ed25e8d0861)

##### VPC

![Screenshot 2023-08-10 at 2 55 45 PM](https://github.com/apache/cloudstack/assets/5229147/77060188-1a94-41db-9e78-0c4b5b03b4c9)

**Comment by weizhouapache** [comment]
added to 4.18.1.0 milestone and assigned to myself

**Comment by weizhouapache** [comment]
@rajujith 
this seems not to be a bug.

the load balancing rules needs to be associated to a network.
You need to choose a VPC tier in the popup, but please do not select any VM

**Comment by rajujith** [comment]
@weizhouapache  but the 'Add VMs' is still a little confusing when we are not expected to add any VM right?

**Comment by DaanHoogland** [comment]
fixed by #7841

**Comment by DaanHoogland** [comment]
fixed by #8793
